tag-expressions/go/parser.go

```diff
@@ -112,14 +112,20 @@ func tokenize(expr string) []string {
 	for _, c := range expr {
 		if unicode.IsSpace(c) {
 			collectToken()
+			escaped = false
 			continue
 		}
 
 		ch := string(c)
 
 		switch ch {
 		case "\\":
-			escaped = true
+			if escaped {
+				token.WriteString(ch)
+				escaped = false
+			} else {
+				escaped = true
+			}
```

Comment:
I would not be opposed to lifting this case out into a separate if-block so that the `escaped = false` in the `unicode.IsSpace(c) {` block is not needed.